db/views.c

```diff
@@ -2631,25 +2635,123 @@ static int _view_update_table_version(timepart_view_t *view, tran_type *tran)
     return rc;
 }
 
-#if 0
-static int bdb_lock_table_write_pp(const char *tblname, timepart_sc_arg_t *arg)
+/**
```

Comment:
Any specific reason for having these systbl changes done here than in sqlite/ext/comdb2/timepartitions.c.

Reply:
Abstraction isolation: I do not want to expose time partition implementation to sqlite library.  
I am also not thrilled about the location of comdb2 sqlite extension modules; most of these are close users of stuff living in db/ folder.